client/apps/edit/components/admin/components/verticals_tags.jsx

```diff
@@ -71,32 +73,32 @@ export class AdminVerticalsTags extends Component {
           <div className='field-group tags'>
             <label>Topic Tags</label>
             <AutocompleteInlineList
-              items={article.get('tags') || []}
+              items={article.tags || []}
               filter={(tags) => {
                 return tags.results.map((tag) => {
                   return { id: tag.id, name: tag.name }
                 })
               }}
               formatSelected={(tag) => tag.name}
-              onSelect={(tags) => onChange('tags', tags)}
+              onSelect={(tags) => onChangeArticleAction('tags', tags)}
               placeholder='Start typing a topic tag...'
-              url={`${sd.API_URL}/tags?public=true&q=%QUERY`}
+              url={`${apiURL}/tags?public=true&q=%QUERY`}
             />
           </div>
 
           <div className='field-group tracking-tags'>
             <label>Tracking Tags</label>
             <AutocompleteInlineList
-              items={article.get('tracking_tags') || []}
+              items={article.tracking_tags || []}
```

Comment:
Not sure if `AutocompleteInlineList` is a custom component, but to make safer overall maybe `AutocompleteInlineList.defaultProps = { items: [] }`?